.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Report a problem with this library that needs to be fixed.
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Before filing this issue**
+Make sure you've carefully reviewed the
+[documentation](https://developers.google.com/google-ads/api/docs/client-libs/python)
+for this library.
+
+This library is only used for the
+[Google Ads API](https://developers.google.com/google-ads/api/docs/start), for
+the [AdWords API](https://developers.google.com/adwords/api/docs/guides/start)
+client library report your issue
+[here](https://github.com/googleads/googleads-python-lib/issues).
+
+Only use this page to report problems with this library. If you are unsure
```

Comment:
"with this library that are specific to Python."

Reply:
"specific to Python" seems redundant, do you mean to differentiate from the other libraries?

Reply:
See my other question.

Reply:
I think it is worthwhile to be extra explicit that issues here are specific just to this client library written in Python and that anything else is out of scope

Reply:
Done